packages/big-design/src/components/Alert/__snapshots__/spec.tsx.snap

```diff
@@ -330,7 +330,7 @@ exports[`render warning Alert 1`] = `
 }
 
 .c1 {
-  grid-gap: 1rem;
+  gap: 1rem;
   display: grid;
```

Comment:
not a biggy, but noticed that some components use the `grid-gap` internally, should we update it to the new name `gap` css prop? If so, this change should be made in another PR?

Example:
```jsx
export const StyledInlineMessage = styled(Grid)<InlineMessageProps>`
  border: ${({ theme }) => theme.border.box};
  border-radius: ${({ theme }) => theme.borderRadius.normal};
  grid-gap: ${({ theme }) => theme.spacing.xSmall};
  grid-template-columns: ${({ theme }) => `${theme.spacing.large} 1fr ${theme.spacing.large}`};
  padding: ${({ theme }) => theme.spacing.xSmall};
`
``` 

```jsx
export const StyledInlineMessage = styled(Grid)<InlineMessageProps>`
  border: ${({ theme }) => theme.border.box};
  border-radius: ${({ theme }) => theme.borderRadius.normal};
  gap: ${({ theme }) => theme.spacing.xSmall};
  grid-template-columns: ${({ theme }) => `${theme.spacing.large} 1fr ${theme.spacing.large}`};
  padding: ${({ theme }) => theme.spacing.xSmall};
`
```